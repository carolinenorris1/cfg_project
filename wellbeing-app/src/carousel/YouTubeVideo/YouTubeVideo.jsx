import React from "react";

function YouTubeVideo() {
  return (
    <iframe
      width="500"
      height="675"
      src="https://www.youtube.com/embed/hzUStg_pGBE"
      title="YouTube video player"
      frameBorder="0"
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
      allowFullScreen
    ></iframe>
  );
}

export default YouTubeVideo;
