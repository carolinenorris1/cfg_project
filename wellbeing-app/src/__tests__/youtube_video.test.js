import React from "react";
import renderer from "react-test-renderer";

import YouTubeVideo from "../carousel/YouTubeVideo/YouTubeVideo";

it("renders correctly when user views the youtube video on the carousel", () => {
  const tree = renderer.create(<YouTubeVideo />).toJSON();
  expect(tree).toMatchSnapshot();
});
