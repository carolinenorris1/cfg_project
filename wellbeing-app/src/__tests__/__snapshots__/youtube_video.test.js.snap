// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when user views the youtube video on the carousel 1`] = `
<iframe
  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
  allowFullScreen={true}
  frameBorder="0"
  height="675"
  src="https://www.youtube.com/embed/hzUStg_pGBE"
  title="YouTube video player"
  width="500"
/>
`;
